plone/formwidget/masterselect/widget.py

```diff
@@ -207,6 +208,7 @@ def __call__(self):
         self.request.response.setHeader(
             'Content-Type', 'application/json; charset=utf-8')
 
+        # import pdb; pdb.set_trace()
```

Comment:
@wkbkhard accident? :)

Reply:
whoops - definitely accident. I will update it as soon as possible!